opc-ua-sdk/sdk-client/src/main/java/org/eclipse/milo/opcua/sdk/client/AddressSpace.java

```diff
@@ -1313,6 +1315,23 @@ private UaViewNode newViewNode(NodeId nodeId, List<DataValue> attributeValues) {
         );
     }
 
+    @Nullable
+    private static <T> T getAttributeOrNull(DataValue value, Class<T> attributeClazz) {
+        StatusCode statusCode = value.getStatusCode();
+
+        if (statusCode != null && statusCode.isBad()) {
+            return null;
+        } else {
+            Object attributeValue = value.getValue().getValue();
+
+            try {
+                return attributeClazz.cast(attributeValue);
+            } catch (ClassCastException e) {
+                return null;
```

Comment:
since the value may exist here so, can we try and return the value instead of null.

in my case it was Uint(10) instead of Double so can we cast it to `Double.valueOf(i)` else return null.

@kevinherron

Reply:
I'm not interested in adding logic to every attribute read for coercing values of incorrect types from a defective server into the correct type. It's enough that we don't throw an uncaught exception here any more.